src/main/java/nl/novi/techiteasy/controllers/ExceptionController.java

```diff
@@ -0,0 +1,15 @@
+package nl.novi.techiteasy.controllers;
+
+import nl.novi.techiteasy.exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ExceptionController {
+    @ExceptionHandler(value = RecordNotFoundException.class)
+    public ResponseEntity<Object> exception(RecordNotFoundException exception) {
```

Comment:
jou exception.getMessage() is een string, dus je zou hier ook als return waarde kunnen invullen een ResponseEntity<String>